src/oneD/StFlow.cpp

```diff
@@ -1060,6 +1068,9 @@ void StFlow::evalContinuity(size_t j, double* x, double* rsd, int* diag, double
                 - (rho_u(x,j+1) - rho_u(x,j))/m_dz[j]
                 - (density(j+1)*V(x,j+1) + density(j)*V(x,j));
         }
+    } else {
+        // unstrained
+        rsd[index(c_offset_U, j)] = rho_u(x, j) - rho_u(x, j - 1);
```

Comment:
This is specifically the unstrained case with fixed mass flow rate. The "free flame" case above is also unstrained.